aws/cf_template.go

```diff
@@ -82,6 +83,11 @@ func generateTemplate(spec *stackSpec) (string, error) {
 			Description: "H2 Enabled",
 			Default:     "true",
 		},
+		parameterLoadBalancerWafWebACLIdParameter: &cloudformation.Parameter{
+			Type:        "String",
+			Description: "WAF Id or ARN",
+			Default:     "",
```

Comment:
This is what I expect to be a problem mentioned in https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/331
Could it be that this ends in the cf stack?

Reply:
I noticed the error while I was testing in my cluster

It looked like master was broken, I fixed in order to unblock my test

Sorry, I should have pointed it here

Reply:
I mean, it failed with an error even when I didn't provide an WAF config